search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -789,3 +798,62 @@
                         deleted-colls-revision-date-validation])]
     (when (seq errors)
       (errors/throw-service-errors :bad-request errors))))
+
+(defn- deleted-grans-result-format-validation
+  "Validates that the only result format support by deleted granules search is :xml"
+  [params]
+  (when-not (= :json (:result-format params))
+    [(format (str "Result format [%s] is not supported by deleted granules search. "
+                  "The only format that is supported is json")
+             (name (:result-format params)))]))
+
+(defn- unrecognized-deleted-grans-params-validation
+  "Validates that no invalid parameters were supplied to deleted granules search"
+  [params]
+  (map #(format "Parameter [%s] was not recognized." (csk/->snake_case_string %))
+       (set/difference (set (keys params)) valid-deleted-granules-search-params)))
+
+(defn- validate-deleted-grans-revision-date-str
+  [revision-date]
+  (when (.contains revision-date ",")
+    [(format (str "Invalid format for revision date, only a starting date is allowed "
+                  "for deleted granules search, but was [%s]") revision-date)]))
+
+(defn- deleted-grans-revision-date-validation
+  "Validates that deleted time can only have a start date for deleted collections search"
+  [params]
+  (when-let [revision-date (or (:revision-date params)
```

Comment:
Similar comment: params should be processed so that we only need to look for the dash case.

Reply:
see other comment, this works the same as deleted collections does.  I was trying to be consistent.